### PR TITLE
Added 'ident' and 'passthrough' options for policy plugin

### PR DESCRIPTION
**Pull request by UladzimirTrehubenka**

### CODE REVIEW

File: contrib/coredns/policy/policy.go [hunk 1]
```diff
@@ -140,6 +142,12 @@ func (p *policyPlugin) parseOption(c *caddy.Controller) error {
 
 	case "transfer":
 		return p.parseTransfer(c)
+
```
Comment (vasili-v) [inline contrib/coredns/policy/policy.go]: I think it's time to update README with all new options.

File: contrib/coredns/policy/policy.go [hunk 2]
```diff
@@ -466,8 +481,14 @@ func (p *policyPlugin) ServeDNS(ctx context.Context, w dns.ResponseWriter, r *dn
 		sendExtra bool
 	)
 
-	debugQuery := p.decodeDebugMsg(r)
 	qName, qType := getNameType(r)
+	for _, s := range p.passthrough {
+		if strings.HasSuffix(qName, s) {
+			return plugin.NextOrFailure(p.Name(), p.next, ctx, w, r)
```
Comment (rdrozhdzh) [inline contrib/coredns/policy/policy.go]: Hm, what should we report with DNStap this case?

Comment (UladzimirTrehubenka) [reply contrib/coredns/policy/policy.go]: I think no

File: contrib/coredns/policy/policy.go (hunk 1 above)
Comment (UladzimirTrehubenka) [reply contrib/coredns/policy/policy.go]: Updated

File: contrib/coredns/policy/README.md [hunk 3]
```diff
@@ -25,18 +33,30 @@ Param START and END (last data byte index + 1) allow to get separate part of edn
 
 Option debug_query_suffix SUFFIX (should have dot at the end) enables debug query feature.
 
+Option streams set gRPC streams count for PDP connection.
+
+Option transfer defines set of attributes (from domain validation response) that should be inserted into IP validation request.
+
+Option ident set string that is used for debug query response as unique id for determine what CoreDNS instance replies on the request.
```
Comment (vasili-v) [inline contrib/coredns/policy/README.md]: Maybe `debug-id`?

**Comment by vasili-v** [comment]
👍

**Pull request merged**